modules/sdk-unified-wallet/src/ecdsa-unified-wallet/ecdsaUnifiedWallets.ts

```diff
@@ -1,16 +1,22 @@
-import { GenerateWalletOptions, BitGoBase, KeychainsTriplet, SupplementGenerateWalletOptions } from '@bitgo/sdk-core';
-import { UnifiedWalletKeychains } from '../unifiedWalletKeychains';
+import {
+  GenerateWalletOptions,
+  BitGoBase,
+  KeychainsTriplet,
+  SupplementGenerateWalletOptions,
+  Keychains,
+} from '@bitgo/sdk-core';
 import { UnifiedWalletID, UnifiedWallet, GenerateUnifiedWalletOptions } from '../types';
 import { supportedCoins } from './types';
 import { UnifiedWallets } from '../unifiedWallets';
 
 export class EcdsaUnifiedWallets extends UnifiedWallets {
-  constructor(bitgo: BitGoBase) {
-    super(bitgo);
+  constructor(bitgo: BitGoBase, coinName = 'eth') {
```

Comment:
The only reason this is there is for testing purposes (teth) but if it wasn't for that it would just not be exposed. The reason is we just need an 'exemplar' ecdsa coin to generate the keys. Wallet generation happens after ecdsa keys are done being generated. We then used the same set of key (from the examplar coin) to create wallets for each of the coins we want (the `supportedCoins`)